### ISSUE TITLE
ignores are ignored

### ISSUE THREAD

**Issue by ns-rse** [DavidAnson/markdownlint-cli2#222]
I have the following simple configuration in YAML following the conventions in the [example](https://github.com/DavidAnson/markdownlint-cli2/blob/main/test/markdownlint-cli2-yaml-example/.markdownlint-cli2.yaml).

I wish to lint `*.md` and `*.markdown` files across all directories but ignore all `*.svg` files regardless of their location.

```yaml
# Configuration
config:
  # MD013 - line-length
  line_length:
    line_length: 120
    code_blocks: false
    tables: false
  html:
    allowed_elements:
      - div

# Globs
globs:
  - "**/*.md"
  - "**/*.markdown"

# Ignores
ignores:
  - "*.svg"

# Fix any fixable errors
fix: true
```

I then create a dummy `test.svg` with trailing lines add and commit (I'm using `markdownlint-cli2` via `pre-commit`) and the trailing lines are then removed when the file should be ignored.

```bash
❱ git commit -m "Test"
trim trailing whitespace.............................(no files to check)Skipped
fix end of files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook

Fixing test.svg

check yaml...............................................................Passed
markdownlint-cli2....................................(no files to check)Skipped
```

I've tried using a glob of `- "**/*.svg"` instead but with the same result.

I can't work out what I'm doing wrong here. :confounded:

**Comment by DavidAnson** [comment]
According to the output you show, the edit to that SVG file is being done by some OTHER tool you have configured in pre-commit. Based on the output, it looks like the "fix end of files" task. This tool doesn't run until after that edit has already been made.

(Also FYI, you have this tool configured to only look at md and markdown files, so telling it to ignore SVG files is unnecessary anyway.)

**Comment by ns-rse** [comment]
:facepalm: I'm an idiot for not spotting that, sorry and thank you.

Also for pointing out that ignores are not required because of the includes.
